rosette/rbl/rosette/consume.rbl

```diff
@@ -0,0 +1,111 @@
+;;; File containing the reflective method consume and its helper methods
```

Comment:
https://www.gnu.org/licenses/gpl-howto.html

I just want to follow the GPL rules, because that's what everyone else does. Each file should reference the copyright.

Reply:
I don't know why Google quit putting copyright in each file, but I know that legally it's not essential; it just mitigates risks where people look at one source file without looking at the LICENSE file at the top of the project (which happens quite frequently when source files are browseable via the web).

Legal theory aside, project conventions are important too.